ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -52,6 +52,20 @@ def _is_pcs_cluster_running(self):
             return False
         return True
 
+    def _wait_for_node_online(self, nodeid: str) -> bool:
+        """
+        Wait till the node becomes online
+        """
+        for retry_index in range(0, const.CLUSTER_RETRY_COUNT):
```

Comment:
Currently we are waiting for max 30 sec. - 5*6. Is it possible that this time increases with the number of nodes in the cluster?

Reply:
I guess so @ajaysrivas.

Reply:
Can we create a github issue or jira ticket to experiment and if required, use number of nodes in place of CLUSTER_RETRY_COUNT.

Reply:
Created https://jts.seagate.com/browse/EOS-20642 @ajaysrivas